### PR TITLE
Update example LVGL_Arduino.ino for v9

### PR DESCRIPTION
**Pull request by danie1kr**
### Description of the feature or fix

* Move buffer to global variables
* lv_tick_inc so LVGL recognizes the passed time

### CODE REVIEW

File: examples/arduino/LVGL_Arduino/LVGL_Arduino.ino [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
 /*LVGL draw into this buffer, 1/10 screen size usually works well. The size is in bytes*/
 #define DRAW_BUF_SIZE (TFT_HOR_RES * TFT_VER_RES / 10 * (LV_COLOR_DEPTH / 8))
+uint8_t draw_buf[DRAW_BUF_SIZE];
```
Comment (XuNeo) [inline examples/arduino/LVGL_Arduino/LVGL_Arduino.ino]:
```suggestion
static uint32_t draw_buf[DRAW_BUF_SIZE / 4];
```

And update below code to 
```c
disp = lv_tft_espi_create(TFT_HOR_RES, TFT_VER_RES, draw_buf, sizeof(draw_buf));
```

Comment (danie1kr) [reply examples/arduino/LVGL_Arduino/LVGL_Arduino.ino]: updated. `uint32_t` might be easier to read when debugging :D

**Review by XuNeo — APPROVED** [review]

File: examples/arduino/LVGL_Arduino/LVGL_Arduino.ino (hunk 1 above)
Comment (XuNeo) [reply examples/arduino/LVGL_Arduino/LVGL_Arduino.ino]: Yes. And it's also required by the software renderer.

**Comment by danie1kr** [comment]
Thanks!

**Comment by kisvegabor** [comment]
We thank you :slightly_smiling_face:

**Comment by LeCyberDucky** [comment]
Hi, could somebody explain the switch from `lv_timer_handler` to `lv_task_handler`?

**Comment by danie1kr** [comment]
I think I saw this in another file/example and thought it is a move for the v9 API. As it just calls `lv_timer_handler` I thought it cannot do much harm 😇

**Pull request merged**